db/migrate/20230614110143_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] In the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the many `Comments` belong to A `Post`. To illustrate that, you need to add a `post` ref to the `comments` table. The post ref will help to map all comments created to a particular post in the database. Kindly implement this. 👍 💻